### DISCUSSION TITLE
Request entity too large

### DISCUSSION THREAD

**Discussion by AntoineGGG** [GrapesJS/grapesjs#4291] [Q&A]
Hey guys, I'm almost done with my work on grapesJS, but get an issue with the autosave and my req.body size.

When I store the editor Content the req.body contain:

gjs-components
gjs-assets
gjs-css
gjs-style
gjs-html

The problem is, I think, when send all of them into the request, the body is too large for Node/Express.
Is there a way to solve this ? 

I don't paste the gjs-assets lines here cause too long but I think you can see what I want to mean.
The thing is I load asset with a custom method when the component is rendered, so I don't need to save the gjs-assets.
Is there a way to use beforeSend to remove the gjs-assets line ?  Or something similar ? 

I already tried to use this on my back-end: 
`app.use(
  bodyParser.urlencoded({
    extended: true,
    limit: "50mb",
    parameterLimit: 1000000000000,
    type: "application/json",
  })
);
app.use(express.json({ limit: "50mb" }));
app.use(express.urlencoded({ limit: "50mb" }));`

Thanks a lot

**Comment by AntoineGGG** [comment]
For now I fix this using:

`fetchOptions: (opts) => {
    if (opts.body) {
      const contentWithoutAssets = JSON.parse(opts.body);
      delete contentWithoutAssets["gjs-assets"];
      opts.body = JSON.stringify(contentWithoutAssets);
    }
    return opts.method === "post" ? { method: "PATCH" } : {};
  },`

But not sure it's the proper way to do it...

**Reply by ronaldohoch** [reply]
How are you getting the editor code? With `editor.getHtml()` 
![image](https://user-images.githubusercontent.com/2287371/169114913-2bcbe762-d1c5-4d89-ab45-86869c2b8382.png)

**Reply by AntoineGGG** [reply]
Nop with the storage manager, I don't use the getHtml() function.
